content/en/blog/2023/istio-at-kubecon-na/index.md

```diff
@@ -0,0 +1,128 @@
+    >}}
+
+The Contribfest session on "Istio Hands-on Development and Contribution Workshop" by Lin Sun, Eric Van Norman, Steven Landow, and Faseela K was also well received. It was great to see so many people interested in contributing to Istio and pushing their first Istio Pull Request at the end of the workshop.
+
+{{< image width="75%"
+    link="./contrib-fest.jpg"
+    caption="KubeCon NA 2023, Contribfest Istio Workshop"
+    >}}
+
+The much awaited panel discussion on "Service Mesh Battle Scars: Technology, Timing and Tradeoffs" which was led by the maintainers from the leading CNCF Service Mesh projects, had a huge crowd in attendance and a lot of interesting discussions.
+
+{{< image width="75%"
+    link="./servicemesh-battle-scars-panel.jpg"
+    caption="KubeCon NA 2023, Service Mesh Battle Scars!"
+    >}}
+
+The following sessions at KubeCon North America were based on Istio and almost all of them had very good content:
+* [Take It to the Edge: Creating a Globally Distributed Ingress with Istio & K8gb](https://sched.co/1R2o5/)
+* [Contribfest: Istio: Hands-on Development and Contribution Workshop](https://sched.co/1R2q7/)
+* [Istio: The Past, Present and Future of the Project and Community](https://sched.co/1R2tA)
+* [Under the Hood: Exploring Istio's Lock Contention and Its Impact on Expedia's Compute Platform](https://sched.co/1R2uV)
+* [Service Mesh Battle Scars: Technology, Timing, and Tradeoffs](https://sched.co/1R2ts)
+* [Untangling Your Service Mesh with Feature Gates](https://sched.co/1R2v6)
+* [Flavors of Certificates in Service Mesh: The Whys and Hows!](https://sched.co/1R2wC)
+
+Istio had a fulltime kiosk in the project pavilion at KubeCon + CloudNativeCon North America 2023, with the majority of questions asked around the schedule for ambient mesh being production ready.
+
+{{< image width="75%"
+    link="./istio-booth-1.jpg"
+    caption="KubeCon NA 2023, Istio Kiosk, Interesting Discussions"
+    >}}
+
+We are glad that the major question which we had at the Istio kiosk in Europe about *CNCF graduation* is already answered, and we assure everyone that we are working on the ambient mesh promotion with the same level of seriousness. Many of our members and maintainers offered support at the booth, helping us answer all the questions from our users.
+
+{{< image width="75%"
+    link="./istio-booth-2.jpg"
+    caption="KubeCon NA 2023, Our maintainers at the Istio Kiosk"
+    >}}
+
+Another highlight about the Istio kiosk this time was that we had our new Istio T-shirts sponsored by Microsoft, Solo.io, Tetrate, and Stackgenie for everyone to grab.
+
+{{< image width="75%"
+    link="./istio-t-shirts.jpg"
+    caption="KubeCon NA 2023, The cool new Istio T-shirts"
+    >}}
+
+We would like to express our heartfelt gratitude to our platinum sponsors Google Cloud, for supporting Istio Day North America! Last but not least, we would like to thank our Istio Day Program Committee members for all their hard work and support!
```

Comment:
Would it be possible to list the members here or link to a listing that already exists?